src/Pages/Music/Albums.elm

```diff
@@ -43,27 +42,51 @@ type alias Params =
 type alias Model =
     { album_list : List AlbumObj
     , route : Route
+    , currentButton : Selected
```

Comment:
I think we want a List of `Selected`, but selected should be changed too

Reply:
It's not clear to me why we need a list of selected here. The currentButton should hold which button is currently in use / was clicked.

Reply:
I would role the info of the bools below into this. Is the information which was the last one the user interacted with really needed?

Reply:
Okay checking the old interface shows we don't need a list. But then we shouldn't also need to save a bool (or better Asc/Desc) fore each one, just one should be enough.